src/GoogleFit/FitOnFhir.GoogleFit/Services/UsersService.cs

```diff
@@ -144,5 +152,20 @@ public override async Task RevokeAccessRequest(PlatformUserInfo platformUserInfo
                 platformUserInfo.ImportState = DataImportState.Unauthorized;
             }
         }
+
+        private async Task RedirectAuthorization(AuthState state, CancellationToken cancellationToken)
+        {
+            var query = new Dictionary<string, string>()
+            {
+                [Constants.StateQueryParameter] = state.State,
+            };
+            var uri = QueryHelpers.AddQueryString(state.RedirectUrl, query);
+            var response = await _httpClient.SendAsync(new HttpRequestMessage(HttpMethod.Get, uri), cancellationToken);
```

Comment:
I'm not sure we want to make this request, I think we need to update the `ProcessAuthorizationCallback()` method to return this uri and the `GoogleFitAuthorizationHandler` should then return a `RedirectResult`.

This is because we need the client (a browser or a mobile application) to actually call the redirect uri. The way the code is written now, the azure function will use the client to call the redirect uri, but we will just return a 200 to the browser. This will result in our function getting Rick Rolled, and not the user.